amundsen_application/static/js/components/Feedback/FeedbackForm/styles.scss

```diff
@@ -1,65 +1,46 @@
 @import 'variables';
-
-.submit {
-  float: right;
-  margin-top: 16px;
-}
-
-.submit:hover {
-  background-color: $gray-lighter;
-}
-
-.radio-set {
-  display: flex;
-  margin-top: 8px;
-}
-
-.radio-set-item {
-  cursor: pointer;
-}
-
-.radio-set .radio-set-item:not(:first-child) {
-  margin-left: 12px;
-}
-
-.radio-set-item label {
-  width: 100%;
-  text-align: center;
-}
-
-.nps-label {
-  font-family: $font-family-header;
-  font-weight: $font-weight-header-regular;
-  margin-bottom: 15px;
-  width: 65px;
-}
-
-.status-message {
-  font-family: $font-family-header;
-  font-weight: $font-weight-header-regular;
-  text-align: center;
-  position: absolute;
-  font-size: 20px;
-  color: $text-medium;
-  /* for centering when parent has automatic height */
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  width: 100%;
-}
-
-input[type="radio"] {
```

Comment:
What does the form look like with these sets of styles removed?

Reply:
These styles have been applied to inputs site-wide and have been moved to `_inputs.scss`.

Reply:
Ah wait, might also be worth mentioning some of these may have to end up back in the feedback component specifically. For example the `margin: 5px;` on radio buttons -- if i remember correctly -- was just for the purposed of getting the visual spacing we  wanted for laying out the 10 horizontal radios for the rating. No need to modify this now, we can update if/when we see any undesired affect on other radio buttons.